libraries/common/tests/test_graph.py

```
@@ -525,6 +525,45 @@ async def run(conn: typing.Any) -> str:
+            return 'ok'
+
+        with self.assertLogs(client.LOGGER, level='WARNING'):
+            result = await g._run_retrying_poisoned(run)
+        self.assertEqual('ok', result)
+        self.assertEqual([conns[0], conns[1]], seen)
+        conns[0].close.assert_not_awaited()
+
+    async def test_other_internal_errors_do_not_retry(self) -> None:
+        g = graph.Graph()
+        g.opened = True
+        pool, conns = _mock_pool_conns(2)
```

Ruff (0.16.1)

[warning] Unpacked variable `conns` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)